enacts/wat_bal/maproom_monit.py

```diff
@@ -459,14 +470,19 @@ def wat_bal_tile(tz, tx, ty):
     )
 
     mymap_min = 0
-    mymap_max = 60 #taw.max()
+    mymap_max = 8* (int(taw_max.values) // 8)
```

Comment:
I had to compute taw_max at the top of the file otherwise I would be returned an error that a resource is already in use. That's probably all fine but I'd figure I'd mention that here as that goes along the lines of the things I don't quite understand in this PR.

Reply:
Per my comment on that discussion you pointed to, then it would make sense that I put this taw_max hard-coded in config file.